Documentation for setting up and moving to the remaining marketplaces (#1901)
aiven/devportal: docs/platform/howto/move-to-gcp-marketplace-billing.rst

**From your new Aiven organization with the GCP marketplace subscription:**

> * Your new Aiven organization name, as shown at the top right of the `Aiven GCP console <https://console.gcp.aiven.io/>`_.

Reviewer: ```suggestion
* Your new Aiven organization name, as shown in the `Aiven GCP console <https://console.gcp.aiven.io/>`_.
```

Reviewer: In general, we try to avoid using directional language for accessibility and because this can change quite a lot.
